modules/ROOT/partials/misc/inline-formatting-of-list-bullets.adoc

```diff
@@ -0,0 +1,24 @@
+== Inline formatting
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+List bullets and list numbers can now be inline-formatted.
```

Comment:
```suggestion
List bullets and list numbers can now be inline-formatted with formats that are possible to apply list bullets or numbers.
```

There is only fixed set of formats that gets applied here any formats that have any of these style properties:
font-weight, font-style, color, font-size, font-family

So bold, italic, text color, font size and font family formats can be applied. Anything else is not expanded to the LI elements.

Not sure how well we need to describe this or how much detail. We can always expand on this if it becomes confusing in support etc. But I think we should at least indicate that it's not all inline formats.

- [ ] Consider having details about that we only apply formats that can apply to list bullets/numbers

Reply:
Added a sentence the notes only a sub-set of all inline-formatting options are supported. The new sentence is third in the section for the sake of information/narrative flow.

1. what we can now do. (tell ’em)
2. what, specifically, this is.
3. re-state what we can do with appropriate qualifications and parameters. (tell ’em you told ’em)
4. how to do it. (tell ’em again)